src/pages/home/report/ReportActionItemMessage.tsx

```diff
@@ -57,7 +57,7 @@ function ReportActionItemMessage({action, displayAsGroup, reportID, style, isHid
         const originalMessage = action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU ? action.originalMessage : null;
         const iouReportID = originalMessage?.IOUReportID;
         if (iouReportID) {
-            iouMessage = ReportUtils.getReportPreviewMessage(ReportUtils.getReport(iouReportID), action);
+            iouMessage = ReportUtils.getIOUReportActionDisplayMessage(action);
```

Comment:
@Gonals Why was this changed? I think it caused this deploy blocker https://github.com/Expensify/App/issues/34685